File: BUBBLE_SORT.py
def BUBBLE_SORT():
    # BUBBLE SORT
    m = []
    for i in range(int(input("Enter length of your list : "))):
        a = int(input(f"Enter element {i+1} : "))
        m.append(a)
    n = len(m)
    for i in range(n):
        for j in range(0, n-i-1):
            if m[j] > m[j+1]:
                m[j], m[j+1] = m[j+1], m[j]
    print(m)


BUBBLE_SORT()

File: Bubble_Sort.java
import java.util.*;

public class Bubble_Sort {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        // Array input
        System.out.print("Enter length of array : ");
        int n = sc.nextInt();
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            System.out.print("Enter element " + (i + 1) + " : ");
            arr[i] = sc.nextInt();

        }
        System.out.println();
        // Printing original array
        System.out.print("Original array : ");
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }

        // array bubble sort
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < n - i - 1; j++) {
                if (arr[j] > arr[j + 1]) {
                    // swapping
                    int temp = arr[j];
                    arr[j] = arr[j + 1];
                    arr[j + 1] = temp;
                }
            }
        }

        // Printing sorted array
        System.out.print("\nSorted array : ");
        for (int i = 0; i < n; i++) {
            System.out.print(arr[i] + " ");
        }

        sc.close();

    }
}
